eventually/static/src/containers/curriculum/CurriculumLink.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
 import {withRouter} from 'react-router-dom';
 import {getTopicListService} from './CurriculumService';
-import {TopicList, TopicDialog} from 'src/containers';
```

Comment:
rename file TopicList to TopicsList so that it has the same name as React component

Reply:
Done